llvm/lib/Target/LoongArch/LoongArchISelLowering.cpp

```diff
@@ -214,6 +214,13 @@ LoongArchTargetLowering::LoongArchTargetLowering(const TargetMachine &TM,
       setOperationAction(ISD::FRINT, MVT::f64, Legal);
   }
 
+  setOperationAction(ISD::FP16_TO_FP,        MVT::f32,   Expand);
+  setOperationAction(ISD::FP_TO_FP16,        MVT::f32,   Expand);
```

Comment:
Please shrink those whitespaces; I believe that's what triggered the style checker. (After the transition away from Phabricator/Arcanist workflow, you have to set up your own auto-formatting and/or pre-commit linting hooks.)